### PR TITLE
Add missing docs in new EnsureCapacity methods

### PR DESCRIPTION
**Pull request by eiriktsarpalis**
Fixes https://github.com/dotnet/runtime/issues/54869. Related to https://github.com/dotnet/runtime/pull/58007.

### CODE REVIEW

File: xml/System.Collections.Generic/List`1.xml [hunk 1]
```diff
@@ -1421,7 +1421,7 @@
         <param name="capacity">The minimum capacity to ensure.</param>
         <summary>Ensures that the capacity of this list is at least the specified <paramref name="capacity" />.
             If the current capacity of the list is less than specified <paramref name="capacity" />, the capacity is increased by continuously twice current capacity until it is at least the specified <paramref name="capacity" />.</summary>
```
Comment (gewarren) [inline xml/System.Collections.Generic/List`1.xml:1423]:
```suggestion
            If the current capacity of the list is less than <paramref name="capacity" />, it is successively increased to twice the current capacity until it is at least the specified <paramref name="capacity" />.</summary>
```

File: xml/System.Collections.Generic/Stack`1.xml [hunk 2]
```diff
@@ -641,7 +641,7 @@
         <param name="capacity">The minimum capacity to ensure.</param>
         <summary>Ensures that the capacity of this Stack is at least the specified <paramref name="capacity" />.
             If the current capacity of the Stack is less than specified <paramref name="capacity" />, the capacity is increased by continuously twice current capacity until it is at least the specified <paramref name="capacity" />.</summary>
```
Comment (gewarren) [inline xml/System.Collections.Generic/Stack`1.xml:643]:
```suggestion
            If the current capacity of the stack is less than <paramref name="capacity" />, it is successively increased to twice the current capacity until it is at least the specified <paramref name="capacity" />.</summary>
```

File: xml/System.Collections.Generic/Queue`1.xml [hunk 3]
```diff
@@ -735,7 +735,7 @@
       <Docs>
         <param name="capacity">The minimum capacity to ensure.</param>
         <summary>Ensures that the capacity of this Queue is at least the specified <paramref name="capacity" />.</summary>
```
Comment (gewarren) [inline xml/System.Collections.Generic/Queue`1.xml:737]:
```suggestion
        <summary>Ensures that the capacity of this queue is at least the specified <paramref name="capacity" />.</summary>
```

**Review by gewarren — APPROVED** [review]

**Comment by eiriktsarpalis** [comment]
@gewarren apologies, it looks like automerge kicked in after you reviewed without taking your unresolved comments into account.

**Comment by gewarren** [comment]
> @gewarren apologies, it looks like automerge kicked in after you reviewed without taking your unresolved comments into account.

No worries, I realized that's what happened. It's unfortunate that automerge doesn't take the comments into account.

**Pull request merged**